plugins/playlist/src/components/CreatePlaylistButton/CreatePlaylistButton.tsx

```diff
@@ -49,11 +55,16 @@ export const CreatePlaylistButton = () => {
       try {
         const playlistId = await playlistApi.createPlaylist(playlist);
         navigate(playlistRoute({ playlistId }));
+        alertApi.post({
+          message: `Added playlist '${playlist.name}'`,
+          severity: 'success',
+          display: 'transient',
+        });
```

Comment:
Can you also add this success message [here](https://github.com/backstage/backstage/blob/master/plugins/playlist/src/components/EntityPlaylistDialog/EntityPlaylistDialog.tsx#L134) as well 🙏 ? this is for when new playlists are created from the entity context